shopStore/ShopStore.js

```diff
@@ -1,26 +0,0 @@
-const RedisStrategy = require('./RedisStrategy');
```

Comment:
no need for this file, no need for env vars since we can just pass in a strategy at will